svthalia/Reaxit
# 406 implement themeable adaptive icons

Closes #406 
### Summary
I changed the icon to an adaptive version by adding a monochrome drawable to `ic_launcher.xml`.

One issue with the adaptive icon is that developers with this feature enabled can no longer distinguish the testing app from the real app. To solve this, I rotated the debug app's icon by 180 degrees (with permission from Jaap) (also increases usability for Australian users).

### How to test
Steps to test the changes you made:
1. Have a fairly recent android device
2. Go to 'Wallpaper & style' in the device settings
3. Enable 'Themed icons'
4. Now the icon should use the device's theme colors
